packages/v4/patternfly-docs/content/design-guidelines/components/inline-edit/inline-edit.md

```diff
@@ -18,7 +18,7 @@ Elements vary depending on inline edit variation
 ### When not to use
 * The editing is the primary function of the view. For example, in an edit modal.
 
-<img src="./img/inline-edit.gif" width="960" height="540" />.
+<img src="./img/inline-edit.gif" alt="Inline edit elements" width="960" height="540" />.
```

Comment:
Rather than "elements", we'd want to use "components" when referring to one of the PatternFly components.

Reply:
Thank you for your feedback! Made a commit below that that took in your feedback!